zerver/views/webhooks/beanstalk.py

```diff
@@ -38,6 +42,7 @@ def _wrapped_view_func(request, *args, **kwargs):
 @has_request_variables
 def api_beanstalk_webhook(request, user_profile,
                           payload=REQ(validator=check_dict([]))):
+    # type: (HttpRequest, UserProfile, Dict) -> HttpResponse
```

Comment:
The Dict here can be more completely annotated as `Dict[str, Any]`; I'll just change that (and the above) and merge.